tools/base/fpgad/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ target_link_libraries(fpgad
   bitstream
   fpgad-api
   dl
+  pthread
```

Comment:
Remove this.  It is included in ${CMAKE_THREAD_LIBS_INIT}.

Reply:
fixed